CHANGES

```diff
@@ -2,6 +2,13 @@
 Waffle Changelog
 ================
 
+v3.0.0
+======
+- Update Github actions
+- Pluggable Switch and Sample models (fix auto-creating custom switches)
```

Comment:
Author note: I don't actually know what this means -- I'm just copying the PR and issue titles. :-)